tor/core/admin_commands.py

```diff
@@ -44,38 +44,38 @@ def process_command(reply, config):
         except KeyError:
             if from_moderator(reply, config):
                 reply.reply(
-                    "That command hasn't been implemented yet ):"
-                    "\n\nMessage a dev to make your dream come true."
+                    'That command hasn\'t been implemented yet ):'
```

Comment:
Extraneous `\` are the other reason that some of these have double quotes and some of them don't.